src/payment/v1/payment-mappers/order-mapper.js

```diff
@@ -50,11 +50,12 @@ export default class OrderMapper {
      * @return {Shipping[]}
      */
     mapToShipping(data) {
-        const { description } = data.shippingOption || {};
+        const { description, transitTime } = data.shippingOption || {};
 
         if (description) {
             return [{
                 method: description,
+                transit_time: transitTime,
```

Comment:
If `description` is null but `transitTime` is defined, do you still want to send `transitTime`? At the moment, it isn't.

Reply:
Great spot 👍  I do want to send `transitTime`